src/controller/python/chip-device-ctrl.py

```diff
@@ -699,7 +750,7 @@ def start_rpc_server():
             print("\nKeyboard interrupt received, exiting.")
             sys.exit(0)
 
-def __get_response_dict(status: StatusCodeEnum, result: Optional[Dict[Any, Any]] = None, error:Optional[str] = None) -> Dict [str, Any]:
+def __get_response_dict(status: StatusCodeEnum, result: Optional[Dict[Any, Any]] = "", error:Optional[str] = "") -> Dict [str, Any]:
```

Comment:
I think we should consider leaving the None in here; and set the allow_none=true on the XMLRPCserver. 

I had the same issue when testing our PASE RPC. 

But I think the API is cleaner if it returns None.

Reply:
Added allow_none=true on RPC Server.